src/app/components/Layout/Header/MenuUser/index.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render } from '@testing-library/react';
+import MenuUser from "./index";
+
+const user = { email: "test@Mail.com" };
+jest.mock("../../../../redux/selectors", () => ({ getAuthUser: () => user }));
+jest.mock("react-redux", () => ({
+    useDispatch: () => () => { },
+    useSelector: (funk) => funk(),
+}));
+
+test("render MenuUser component with user", () => {
+    const screen = render(<MenuUser />);
+    expect(screen.getByText("log out")).toBeInTheDocument();
+    expect(screen.queryByText("log in")).toBeNull();
```

Comment:
better to use `queryByText` and `.not.toBeInTheDocument()` if you check the absence.